mrp_bom_product_details/models/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#########################################################################
```

Comment:
Can you change to short headers? Follow the <a href="https://github.com/OCA/maintainer-tools/tree/master/template/module">template</a>.